spec/pathwatcher-spec.coffee

```diff
@@ -114,6 +114,21 @@ describe 'PathWatcher', ->
           done()
       fs.writeFileSync(tempFile, 'changed')
 
+  describe 'when watching a file that does not exist', ->
+    it 'throws an error with a code', (done) ->
```

Comment:
Ok, I guess I don't know how you differentiate between declaring sync and async tests in this test system. Will try to figure it out.